.gitallowed

```diff
@@ -0,0 +1,5 @@
+MyPASS=${2-"password"}
```

Comment:
These are the strings flagged by secrets checking for `mot-logger` so won't be needed here. Any flagged when committing to this repo? You can also run `git secrets --scan` to check